editor/export/editor_export_platform_apple_embedded.h

```diff
@@ -168,6 +168,9 @@ class EditorExportPlatformAppleEmbedded : public EditorExportPlatform {
 	virtual Error _export_icons(const Ref<EditorExportPreset> &p_preset, const String &p_iconset_dir) { return OK; }
 
 	virtual String get_platform_name() const = 0;
+	virtual String get_sdk_name() const = 0;
+	virtual String get_minimum_deployment_target() const = 0;
```

Comment:
@stuartcarnie @bruvzg Note that I have introduced these two abstract methods. Currently they're used for:
- Submitting the correct `xcodebuild` archive command from `EditorExportPlatformAppleEmbedded::_export_project_helper()`.
- Configuring each additional platform framework (if any) correctly in `EditorExportPlatformAppleEmbedded::_convert_to_framework()`.
- Setting the proper `min_ios_version`/`min_visionos_version` default value for each platform export options.

Reply:
Excellent – makes sense to me!